### ISSUE TITLE
Figure out what still has to done for the image analysis

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/eln.epfl.ch#207]
- [ ] Tine reported a potential bug in the roundness 
- [ ] Tine reported some missing parameters 

One should probably make some case studies to test it in "real-world" scenarios.

**Comment by kjappelbaum** [comment]
fix for issue 1: https://github.com/image-js/image-js/pull/548

**Comment by lpatiny** [comment]
@kjappelbaum  Could you add more information about the bug in roundness ?

**Comment by kjappelbaum** [comment]
>@kjappelbaum Could you add more information about the bug in roundness ?

you merged the fix with https://github.com/image-js/image-js/pull/548 

for the other thing, i'm still waiting for clarification

**Comment by lpatiny** [comment]
Yes I saw the PR after seeing this message ... sorry for the confusion

**Comment by kjappelbaum** [comment]
Do you think we should test with "fake images" or just hard code some simpler things (e.g., hardcoding some parameters that are used in the calculation, like area and fe_max for the roundness). I think I would prefer "fake images", i.e., at least circles. What do you think?

**Comment by lpatiny** [comment]
The ethylbenzene on c6h6 has already a shape image

![image](https://user-images.githubusercontent.com/1484241/105289012-500be900-5bb8-11eb-914b-ae03d379cfca.png)
